lib/core/constants.dart

```diff
@@ -0,0 +1,5 @@
+const maxPasswordLenght = 16;
+const minPasswordLenght = 8;
+const nameMaxLenght = 20;
+const emailMaxLenght = 30;
+const errorState = "Can not reach the server";
```

Comment:
make better name for constant, this is not the state

Reply:
That's already been changed.